app/src/main/java/com/iyanuoluwa/imovie/data/model/buffer/api/Cast.kt

```diff
@@ -1,4 +1,4 @@
-package com.iyanuoluwa.imovie.api2
+package com.iyanuoluwa.imovie.data.model.buffer.api
```

Comment:
Just move this to `model`

```suggestion
package com.iyanuoluwa.imovie.data.model
```